.github/workflows/docker-build.yaml

```
@@ -39,3 +40,20 @@ jobs:
 
     - name: Build bundle docker image
```

Also update this one:

```suggestion
    - name: Build operator bundle docker image
```